assignment3

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <fstream>
+
+int main() {
+
+    //read the file into an array "userinfo"
+    std::ifstream file("config.txt");
+    if(file.is_open()) {
+
+        std::string userinfo[4];
```

Comment:
it would be better to count the file first before you read into it but this works